base_vat_sanitized/models/res_partner.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 Akretion (http://www.akretion.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+
+import re
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    sanitized_vat = fields.Char(
```

Comment:
@alexis-via Even if we know the slowly computation of these kind of fields, shouldn't this be added to commercial_fields ?

https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/models/res_partner.py#L435

Reply:
Nope, because vat is already a commercial field and should be ok on all same partners. I like answering on my own... :smile: